### PR TITLE
[CWS] restrict rule with kill action on policy source

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Allow the kill action on specific rule policy source. By default on from remote-configuration.

### Describe how to test/QA your changes

Try to use a kill action from a local policy file without having the `file` source allowed.

### CODE REVIEW

**Review by YoannGh — APPROVED** [review]

File: pkg/config/setup/system_probe_cws.go [hunk 1]
```diff
@@ -131,4 +131,5 @@ func initCWSSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	// CWS enforcement capabilities
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.enabled", true)
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.raw_syscall.enabled", false)
+	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.rule_source_allowed", []string{"remote-config"})
```
Comment (paulcacheux) [inline pkg/config/setup/system_probe_cws.go:134]: So we do not allow it on files ? This seems surprising

Comment (safchain) [reply pkg/config/setup/system_probe_cws.go:134]: Yes that is the plan for now

**Review by spikat — APPROVED** [review]
The PR LGTM, but same concern as Paul on the default config

File: pkg/config/setup/system_probe_cws.go (hunk 1 above)
Comment (lebauce) [reply pkg/config/setup/system_probe_cws.go:134]: I think we should also allow local policies. It seems strange to not allow a local administrator to use the feature when we can use it over remove config

Comment (safchain) [reply pkg/config/setup/system_probe_cws.go]: I added the `file` source in the default

**Review by lebauce — APPROVED** [review]

**Pull request merged**